MainRepository/Program.cs

```diff
@@ -1,7 +1,19 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find the frequency of alphabets in a dictionary.
+namespace Training {
+   class LetterFrequency {
+      public static void Main () {
+         string[] words = File.ReadAllLines ("c:/etc/words.txt");
+         Dictionary<char, int> dict = new ();
+         foreach (string word in words) {
+            foreach (char ch in word) {
+               if (dict.ContainsKey (ch)) dict[ch] = dict[ch] + 1;

```

Comment:
Use TryGetValue instead of ContainsKey. TryGetValue is just ContainsKey plus an array lookup if the item is present and is faster.
This if else could be simplified with one line if you use TryGetValue.